shell_gpt.sh

```diff
@@ -0,0 +1,125 @@
+ZSH_KEYBINDING='^ '
```

Comment:
This will not work for Mac users, since it will conflict with system wide hotkey for keyboard language change. MacOS just intercepts it, and it doesn't reach the shell/terminal.

Reply:
I thought that was for Command-Space, not Control-Space. I can't find any mention of Control-Space on [keyboard shortcuts](https://support.apple.com/en-us/HT201236).

Reply:
Not sure why it is not there. It seems it [depeneds on version](https://www.google.com/search?q=mac+default+langauge+change+hotkey&oq=mac+default+langauge+change+hotkey&aqs=chrome..69i57j33i10i160j33i22i29i30l2.7271j0j7&sourceid=chrome&ie=UTF-8) of macOS

Reply:
Changed the default binding to Alt+Space. Does this still conflict?

Reply:
Yes, still doesn't work.